.circleci/config.yml

```diff
@@ -49,9 +49,17 @@ jobs:
       - run:
           name: Report coverage
           command: bash <(curl -s https://codecov.io/bash)
+
+      - restore_cache:
+          key: v1-vuln-scans-{{ checksum "yarn.lock" }}
       - run:
           name: Vulnerability Tests
           command: yarn test:vulns
+      - save_cache:
+          key: v1-vuln-scans-{{ checksum "yarn.lock" }}-{{ epoch }}
```

Comment:
Is it okay that we restoring cache without the `epoch` variable?

Reply:
this is necessary because otherwise Circle CI won't update the cache entry if it has existed before (this is its feature). To avoid this 'rewrite ban' the following strategy was suggested: # https://discuss.circleci.com/t/add-mechanism-to-update-existing-cache-key/9014/12

Reply:
As we discussed, please add a small comment with this link before the `key` line 👍

Reply:
agreed to introduce comment for that